### PR TITLE
Fixes the Pub Key error showing in the Install Python2 process.

### PR DESCRIPTION
**Pull request by bwatson78**

### CODE REVIEW

**Comment by bwatson78** [comment]
@rotated8 This change fixes the error, but is the change safe for deployment?

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
 
             - run:
                 name: Install Python2
-                command: sudo apt-get update && sudo apt-get install -y python2
+                command: sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4EB27DB2A3B88B8B && sudo apt-get update && sudo apt-get install -y python2
```
Comment (rotated8) [inline .circleci/config.yml:38]: Does this work if it's split into two lines?
```suggestion
                command: sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4EB27DB2A3B88B8B
                command: sudo apt-get update && sudo apt-get install -y python2
```

**Review by rotated8 — CHANGES_REQUESTED** [review]
Can we split the one line into two?

File: .circleci/config.yml (hunk 1 above)
Comment (rotated8) [reply .circleci/config.yml:38]: Wait, no, that won't work.

Comment (rotated8) [inline .circleci/config.yml:38]:
```suggestion
                command: |
                    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4EB27DB2A3B88B8B
                    sudo apt-get update && sudo apt-get install -y python2
```

**Review by rotated8 — CHANGES_REQUESTED** [review]
I think I fixed the syntax. I think this should work?

File: .circleci/config.yml (hunk 1 above)
Comment (bwatson78) [reply .circleci/config.yml:38]: I'll give it a shot.

Comment (bwatson78) [reply .circleci/config.yml]: Last commit should address your request.

**Comment by rotated8** [comment]
@bwatson78 I think this PR is no longer necessary, correct?

**Comment by bwatson78** [comment]
Right.

**Pull request closed without merging**